playbooks/local_os_client.yaml

```diff
@@ -74,18 +74,41 @@
         set_fact:
           cloudsyaml: "{{ {'clouds': {}} }}"
 
+  - name: Configure cacert locally
+    when: ssl_enabled
+    block:
+    - name: Set local path of cacert
+      set_fact:
+        cacert_path: "{{ ansible_env.HOME }}/.config/openstack/{{ local_cloudname }}-ca.crt"
+
+    - name: Copy CA cert into local config directory
+      copy:
+        dest: "{{ cacert_path }}"
+        content: "{{ hostvars['standalone']['ssl_ca_cert'] }}"
```

Comment:
`ssl_ca_cert` is defined in `playbooks/install_stack.yaml` playbook, I don't think it'll actually work here (you should get an error saying this variable doesn't exist).

Reply:
if we're doing that, do we want to keep `Copy CA cert into PKI` task?

Reply:
We slurped it earlier in this playbook on line 28.